lib/index.js

```diff
@@ -42,63 +40,69 @@ function R(...params) {
 
   const usedSettings = Object.assign({}, defaultSettings, settings);
 
-  return (event, context, callback) => {
+  return async (event, context, callback) => {
```

Comment:
AWS will shortly removed support of node.js 6.10 environment.
As in nodejs 8.10 there was new interface introduced and handler function could return promise, I think, we could have callback param removed and rewrite handler to complete async/await version.
Let's have it in 2.0.0!